installer/roles/kubernetes/defaults/main.yml

```diff
@@ -48,4 +66,4 @@ postgress_activate_wait: 60
 insights_url_base: "https://example.org"
 
 custom_venvs_path: "/opt/custom-venvs"
-custom_venvs_python: "python2"
+custom_venvs_python: "python2"
```

Comment:
Missing newline here.

Reply:
In my local copy there is definitely a new line here.. strange
![image](https://user-images.githubusercontent.com/22608473/63490040-4fb07f80-c4e6-11e9-825a-1b5e7a99f61f.png)